src/main/java/org/apache/commons/lang3/ArrayUtils.java

```diff
@@ -267,15 +268,13 @@ public static boolean isEquals(final Object array1, final Object array2) {
             } else if (object instanceof Object[]) {
                 final Object[] entry = (Object[]) object;
                 if (entry.length < 2) {
-                    throw new IllegalArgumentException("Array element " + i + ", '"
-                        + object
-                        + "', has a length less than 2");
+                    throw IllegalArgumentExceptions.format("Array element %,d, '%s', has a length less than 2", i,
+                            object);
```

Comment:
%,d => %d

Reply:
Hi @sebbASF ,
One of the reasons for the new API is to use rich formatting, so why loose the comma-separated format since it is much easier for humans to read?

Reply:
OK, sorry, I did not realise that the comma was a valid numeric flag for the formatter.
It looked like a mistake.